### DISCUSSION TITLE
collab?

### DISCUSSION THREAD

**Discussion by Co1t0n** [3kh0/ext-remover#361] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hey! I have the same goals as you and want to make a mega-bookmarklet with all the features of my antischool repo and more

**Comment by Aka-but-nice** [comment, marked as answer]
BRO same im making one with the ingot ui rn

**Reply by Aka-but-nice** [reply]
Also like 3kh0 told me, to become a collaborator, you have to get to the top of the most helpful leaderboard, which includes answering questions adding new ways to improve the exploit, and just be the most helpful.

**Reply by rwg100** [reply]
doesnt 3kho literly just select himself for every single post, even in theres a cure for cancer in the replies

**Reply by Aka-but-nice** [reply]
true, but kinda not, like he does include other people just try to answer questions

**Reply by 3kh0** [reply]
@shrimpledimple if you can answer it better, than you will be selected

**Comment by obkduishinijf** [comment]
would be nice, aka is making one rn, and idk if 3kh0 is even helping, or in any way involved with the production of his exploit.

**Reply by Aka-but-nice** [reply]
@aspdfihaeuf he's not, but I don't need his help I'm doing some advanced shi

**Reply by obkduishinijf** [reply]
fair, lmk when a progress update is available, can't wait for it

**Reply by Aka-but-nice** [reply]
Alr its in the progress

**Reply by obkduishinijf** [reply]
W, can't wait

**Reply by Aka-but-nice** [reply]
alr

**Reply by Aka-but-nice** [reply]
so ingot 106 is here

**Reply by Aka-but-nice** [reply]
but its not functional y'all can see the UI

**Reply by Aka-but-nice** [reply]
https://ingot106.glitch.me/ingot.js

**Reply by Aka-but-nice** [reply]
> would be nice, aka is making one rn, and idk if 3kh0 is even helping, or in any way involved with the production of his exploit.

Well I stopped working on the ingot, but I made another one that works for 106 and higher it is listed at https://github.com/3kh0/ext-remover it is a remake of swamp, for most extensions.

**Reply by Aka-but-nice** [reply]
and 3kh0 is, uh was kinda helping, but not really

**Reply by Alex2Mayhem** [reply]
Can i get collab? or no.

**Reply by Aka-but-nice** [reply]
If you start answering questions, I'll mark you with the answer that will help you get to the top of the leader board
